test/src/box/mixed.js

```diff
@@ -19,7 +19,7 @@ describe('mixed registry types', function() {
   // TODO(markdittmer): Come up with a non-manual test that will confirm all
   // registries clean up properly.
 
-  it('should appropriately manage mixed registry types', function(done) {
+  xit('should appropriately manage mixed registry types', function(done) {
```

Comment:
Could you please put a comment in that explains why this test is disabled? Same for the test in `pipeline.js`.

Reply:
ForkBox was removed in the pressure-wash cleanup. Maybe we should also remove the tests that use it.

Reply:
Removed.